GoogleForCreators/web-stories-wp
# Page Templates: Saved page templates

## Summary
- Adds a dropdown for choosing between default and saved templates.
- When switching to "Saved templates", displays a button for saving the current page as a template.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Enable the experiment for saving custom page templates.
2. Open the Page Layouts tab in the library
3. Verify you can choose between the Default and Saved templates
4. Verify the non-functional button appears for Saving page as a template when choosing "Saved templates" from the dropdown
5. Switch off the experiment again
6. Verify that the dropdown is not visible and everything works as before.


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #7058 
Fixes #7059
